src/main/java/teammates/ui/webapi/output/RegenerateStudentCourseLinksData.java

```diff
@@ -0,0 +1,23 @@
+package teammates.ui.webapi.output;
+
+/**
+ * The API output format for the regenerate student's course links request.
+ */
+public class RegenerateStudentCourseLinksData extends ApiOutput {
+    private final String message;
```

Comment:
Do you really need the message (which is like a status message)?

Reply:
It's the message that is shown in the snackbar (like in `MessageOutput`).